### ISSUE TITLE
Change the tooltip for thread scheduler option

### ISSUE THREAD

**Issue by Genjar** [RPCS3/rpcs3#8120]
I've been relying on the tooltips for determining which options to enable. 
I'm using Ryzen 7 2700x, so based on the tooltip I've been keeping the 'Enable thread scheduling' option always enabled. Since it says that it's 'very useful' and has 'huge performance gains' on Ryzen.

Well, couple of days ago I tried to disable it for once, and every game that runs started running far smoother than before. I'm no longer getting constant stuttering every few seconds. So maybe there should be a tooltip warning about how it can make things slower on some systems?

**Comment by kd-11** [comment]
Windows updates are constantly changing behavior of the windows scheduler. Once upon a time, windows scheduler was like 20% the performance of rpcs3's scheduler. Now, different versions of windows give wildly different results, it's becoming difficult to keep track. For older zen, rpcs3's scheduler still has higher peak framerates, but maybe smoothness and pacing is impacted, I don't know for sure.

**Comment by Genjar** [comment]
Here's a couple of example of how much of a difference it makes on my system.

Persona 5:
Thread scheduling disabled: Stable 30fps, with nearly perfect sound. 
Thread scheduling enabled: Generally 30fps, but the intro video is extremely choppy, music stutters throughout the whole game, and the fps drops to less than 5 for several seconds.

Class of Heroes 2:
Disabled: 28-30fps, minor sound crackling, no other issues.
Enabled: 30fps in 2D, choppy sound, fps drops to less than 10 in dungeons, and to less than 5 fps during enemy encounters.

In short, I feel stupid for not trying to disable it sooner..

**Comment by kd-11** [comment]
Do you maybe have 'spu loop detection' enabled as well? Your performance is rather troubling compared to my aging 1700. Your system should be much faster. Can I see your log for p5?

**Comment by Genjar** [comment]
I presume you mean that it's troubling with the scheduler enabled, since without it the games run (as far as I can tell) nearly perfectly. I haven't applied the 60fps patch to P5 yet.

I'm getting best results with SPU loop detection, so I've kept that enabled. Same goes for multithreaded RSX, even though most guides seem to recommend disabling that.

Here's the logs. Played P5 with thread scheduler enabled/disabled up until halfway through the intro. No major differences in the logs as far as I can tell, but in practice.... With thread scheduling disabled it ran perfectly, but when enabled sound was distorted from the start (including the cursor sound in 'do you want to connect to thieves guild' selection) and the intro stalled for the first 20 seconds at about 5 fps. After which it started running smoothly at 30fps for a while. 

[RPCS3 (thread scheduling disabled).log](https://github.com/RPCS3/rpcs3/files/4550879/RPCS3.thread.scheduling.disabled.log)
[RPCS3 (thread scheduling enabled).log](https://github.com/RPCS3/rpcs3/files/4550881/RPCS3.thread.scheduling.enabled.log)

**Comment by vsub** [comment]
Hmm that's weird,your cpu is much more powerful than my i7-6700HQ purposely limited to 2.5Ghz but with the 60FPS patch,I often get 60fps and rarely below 30fps 

Thread scheduler is not enabled and the spurs is set to 5

**Comment by kd-11** [comment]
@Genjar That is not normal behavior with scheduler enabled. On my 1700 the worst performance I get is ~45fps in the backstreets standing in the most demanding spot in the game. Even central street never goes near 40 fps. Audio is smooth, etc, so that stuttery audio in the menus when there is no real work being done is quite alarming. Maybe the scheduler is broken with other CPUs? I'll have to check what it does if I report CPU as 2700x, just in case there is a bug there.

**Comment by kd-11** [comment]
Closing - the scheduler mechanism was overhauled.
